pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -70,7 +70,10 @@ func StartController(config *util.ControllerConfig, stopChan <-chan struct{}) er
 	userAgent := "FederatedTypeConfig"
 	kubeConfig := config.KubeConfig
 	restclient.AddUserAgent(kubeConfig, userAgent)
-	client := fedclientset.NewForConfigOrDie(kubeConfig).CoreV1alpha1()
+	client, err := client.New(kubeConfig, client.Options{})
```

Comment:
(No action required) Consider choosing a variable name that is differentiated from the package name, either by aliasing the import (e.g. `genclient`) or naming the variable differently (e.g. genClient).